pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -847,6 +847,10 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   };
 
+  neotest = super.neorg.overrideAttrs {
```

Comment:
Shouldn't it be `super.neotest.overrideAttrs`?
It looks like a copy paste error from the previous neorg attribute.

I think this is the reason why my nixvim config stopped working.

Reply:
Oh totally right, sorry I missed it in review 🙃.

Reply:
I have just created #269063, if you could look at it, thanks :+1:

Reply:
ooops ! Sorry for that :/
Thanks for the quick fix @ambroisie !

Reply:
@ohlus is the one to thank here, I'm just approving the MR (no commit bit unfortunately).